roles/pulp_devel/tasks/galaxy_ui.yml

```diff
@@ -14,21 +14,11 @@
         path: '{{ developer_user_home }}/devel/ansible-hub-ui'
       register: is_hub_installed
 
-    - name: UI from devel path
-      set_fact:
-        hub_ui_path: '{{ is_hub_installed.stat.path }}'
-      when: is_hub_installed.stat.exists
-
-    - name: UI from git
-      set_fact:
-        hub_ui_path: '{{ pulp_user_home }}/ansible-hub-ui'
-      when: not is_hub_installed.stat.exists
-
```

Comment:
Why are we dropping this feature?

Reply:
not dropping a feture, just ensuring devel env installs at same place:
`{{ developer_user_home }}/devel/ansible-hub-ui`

Reply:
Oh, I see.